### PR TITLE
CLI credentials only generate the LATITUDE_MASTER_KEY. Doesn't include in the build. This improve security.

### PR DESCRIPTION
**Pull request by andresgutgon**
## Describe your changes
CLI credentials only generate the `LATITUDE_MASTER_KEY`. Doesn't included in the build. This improves security. Is left to the person deploying a different master key for the production environment.

## TODO
- [x] Test this is working as expected with Latitude Cloud deploy.

### CODE REVIEW

File: .changeset/spotty-seahorses-sell.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@latitude-data/cli": minor
+---
+
+Allow adding multiple secrets at the same time to Latitude Cloud
```
Comment (geclos) [inline .changeset/spotty-seahorses-sell.md:5]: you are bundling this change in this commit also? would be better in a separate commit

File: docs/views/basics/signed-parameters.mdx [hunk 2]
```diff
@@ -7,12 +7,18 @@ If you need to obfuscate the parameters in the URL, you can sign the parameter `
...
+
 This command will create `LATITUDE_MASTER_KEY` in an `.env` file
 ```sh
 LATITUDE_MASTER_KEY=SOME_SECRET_TOKEN
 ```
+<Warning>It is **crucial** to add your `.env` file to your `.gitignore` before deploying to prevent exposure of your `LATITUDE_MASTER_KEY`.</Warning>
+<Warning>Ensure that you generate a unique `LATITUDE_MASTER_KEY` for the production environment.</Warning>
+
+<Note>If you're using Latitude Cloud, you can create a production key using the command `latitude credentials add`, and then upload it to your Latitude Cloud app environment with `latitude secrets add LATITUDE_MASTER_KEY=<key_generated_by_previous_command>`.</Note>
+
```
Comment (geclos) [inline docs/views/basics/signed-parameters.mdx:21]: this is a lot of notes and warnings in a row does it read ok? usually when you flash readers with many warnings they end up not reading anything

File: packages/cli/src/commands/credentials/index.ts [hunk 3]
```diff
@@ -1,85 +1,12 @@
...
-    status: overwriteKey
-      ? MessageStatus.overwritten
-      : createKey
-        ? alreadyCreated
-          ? MessageStatus.alreadyCreated
-          : MessageStatus.created
-        : secret
-          ? MessageStatus.existing
-          : MessageStatus.missing,
-  })
```
Comment (geclos) [inline packages/cli/src/commands/credentials/index.ts:82]: :feelsgood:

**Review by geclos — APPROVED** [review]
split the change of secrets into a different pr and 🚀

File: .changeset/spotty-seahorses-sell.md (hunk 1 above)
Comment (andresgutgon) [reply .changeset/spotty-seahorses-sell.md:5]: What's the benefit of splitting in 2 PR? The changes are already in different commits

Comment (geclos) [reply .changeset/spotty-seahorses-sell.md:5]: because they get squashed into a single commit when pr is merged. No big deal for this PR since it's a small change but keep it in mind for next times!

File: docs/views/basics/signed-parameters.mdx (hunk 2 above)
Comment (andresgutgon) [reply docs/views/basics/signed-parameters.mdx:24]: Done
image
